readme.md

```diff
@@ -32,10 +32,10 @@ A `build` directory will appear. This is the compiled code that will run in Node
 The default local port for the API to run on is 8000 (`localhost:8000`). If you attempt to run the API and get an error stating that the port is already in use, you may need to change the port listed in the .env file. If you feel that this port should not be taken and an old instance of the API is running still, check the Troubleshooting section below.
 
 ### Sending and Retrieving Contact Data
-- GET: `localhost:8000/contact`
-  - Make a GET request to /contact to see all data that is stored in the API.
-- POST: `localhost:8000/contact`
-  - Make a POST request to /contact to add a contact request to the stored data. This will require a BODY to the POST request that is in the below format:
+- GET: `localhost:8000/contact-requests`
```

Comment:
This change is due to treating the conceptual entity as "contact requests" instead of "contacts", and realizing that change elsewhere.